Test scripts/Ilse/bulk-poli-temp-to-word.vbs

```diff
@@ -73,17 +94,33 @@ Do
         err_msg = ""
         dialog Dialog1
         cancel_without_confirmation
-        If ButtonPressed = select_a_file_button then call file_selection_system_dialog(file_selection_path, ".xlsx")
-        If trim(file_selection_path) = "" then err_msg = err_msg & vbcr & "* Select a file to continue."
-        If err_msg <> "" Then MsgBox err_msg
+        'Initial POLI TEMP List Excel file
+        If ButtonPressed = select_initial_file_button then call file_selection_system_dialog(file_selection_path, ".xlsx")
+        If trim(file_selection_path) = "" then err_msg = err_msg & vbcr & "* Select the initial POLI TEMP List file to continue."
+        'Folder selection where the Word Docs are going to live
+        If ButtonPressed = select_save_folder_button then call select_folder(root_file_path)
```

Comment:
Similar to comment for Excel file selection, if the user selects Browse and then clicks "Ok" on the Browse for Folder dialog, the script immediately runs rather than returning to the initial dialog.